docs/meetings/team/2022_09_27.md

```diff
@@ -0,0 +1,134 @@
+# 🚀 Weekly Team Meeting Minutes
+
+### **Start Time** 9:34pm
+
+### **End Time** 10:30
+
+### **Location** Discord
+
+**Date** Sept 16th 2022
+
+## 👋 Attendence
+
+1. Veronica
+2. Edouard
+3. Matt
+4. Logan
+
+***Attendence Comments***
+
+## 🧸 Check-In
+
+- Dad joke of the day
+
+## Announcements
+
+- Client Meeting needs to be rescheduled/canceled
+- Git Stats due Friday
+- Research before next client meeting
+  
+## Last Week (Restrospective)
+
+### Client meeting
+
+- Important to dicuss with other groups client agenda
+  - Time stamps for each question should become a norm
+  - How much does each person need to get done ?
+- Our group took the meeting over a bit
+- Important to prep questions ahead of time
+  - How do we want to organize the agenda ?
+    - Check in with groups to see how this should get done
+    - Having our own questions would be good and more general ones shared would be okay
+- Languistics, animation, AI/ML, at lot more then orginally thought for the project
+- Thanks matt for bringing the meeting back on track
```

Comment:
This was a huge thing, we might want to discuss this with the other groups today?